spec/facility_spec.rb

```diff
@@ -23,4 +35,101 @@
+      
+      
+      expect(@camaro.registration_date).to eq(Date.today)
+      expect(@camaro.plate_type).to eq(:antique)      
+
+      expect(@facility_1.register_vehicle(@bolt)).to eq([@cruz, @camaro, @bolt])
+      expect(@bolt.registration_date).to eq(Date.today)
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro, @bolt])
+    
+      expect(@facility_1.collected_fees).to eq(325)
+
+
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.services).to eq([])
+      expect(@facility_2.register_vehicle(@bolt)).to eq(nil)
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.collected_fees).to eq(0)
+    end  
+  end
+  
+  describe 'Getting a Drivers Licence' do
+    it 'can input a drivers license' do
+
+  #Written Test
+    expect(@registrant_1.licensed_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    expect(@registrant_1.permit).to eq(true)
+    expect(@facility_1.administer_written_test(@registrant_1)).to eq(false)
+    expect(@registrant_1.licensed_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    expect(@facility_1.add_service('Written Test')).to eq(["Written Test"])
+    expect(@facility_1.administer_written_test(@registrant_1)).to eq(true)
+    expect(@registrant_1.licensed_data).to eq({:written=>true, :license=>false, :renewed=>false})
+    
+    
+    expect(@registrant_2.age).to eq(16)
+    expect(@registrant_2.permit?).to eq(false)
+    expect(@facility_1.administer_written_test(@registrant_2)).to eq(false)
+    @registrant_2.earn_permit
+    expect(@facility_1.administer_written_test(@registrant_2)).to eq(true)
+    expect(@registrant_2.licensed_data).to eq({:written=>true, :license=>false, :renewed=>false})
+    
+    
+    expect(@registrant_3.age).to eq(15)
+    expect(@registrant_3.permit?).to eq(false)
+    expect(@facility_1.administer_written_test(@registrant_3)).to eq(false)
+    
+    @registrant_3.earn_permit
+    expect(@facility_1.administer_written_test(@registrant_3)).to eq(false)
+
+    expect(@facility_1.administer_road_test(@registrant_3)).to eq(false)
+    
+
+# Road Test
+    @registrant_3.earn_permit
+    expect(@facility_1.administer_road_test(@registrant_3)).to eq(false)
+    expect(@registrant_3.licensed_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    expect(@facility_1.administer_road_test(@registrant_1)).to eq(false)
+    expect(@facility_1.add_service('Road Test')).to eq(["Written Test", "Road Test"])
+    expect(@facility_1.administer_road_test(@registrant_1)).to eq(true)
+    expect(@registrant_1.licensed_data).to eq({:written=>true, :license=>true, :renewed=>false})
+    expect(@facility_1.administer_road_test(@registrant_2)).to eq(true)
+    expect(@registrant_2.licensed_data).to eq({:written=>true, :license=>true, :renewed=>false})
+  
+    
+    
+    # Renew License
```

Comment:
These comments are where you could add new it blocks to create smaller, more specific tests.